src/ComparedQueryable/EnumerableRewriter[TComparison].cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using ComparedQueryable.NativeQueryable;
+using Queryable = System.Linq.Queryable;
+
+namespace ComparedQueryable
+{
+    internal class EnumerableRewriter<TComparison> : EnumerableRewriter
+    {
+        private readonly IComparer<TComparison> m_comparer;
+        private static readonly ISet<string> ComparerFunctions = new HashSet<string>(new[]
+        {
+            nameof(Queryable.OrderBy),

```

Comment:
There are only four methods that accept an `IComparer<T>` for `IQueryable<T>` - predictably, the four ordering methods.